bugbug/repository.py

```diff
@@ -109,6 +111,7 @@ def hg_log(hg, first_rev):
             date=dt,
             bug=rev[4],
             ever_backedout=(rev[5] != b''),
+            reviewers=tuple(rev[6].decode('utf-8').split(',')) if rev[6] != b'' else None
```

Comment:
I don't know how long the Mercurial bug will take to get fixed, so let's resort to parsing the commit message for now (do it here though, so we can easily switch to the template later.

Reply:
You mean using regex to get the list of reviewers from the commit message, right? (`r=''`)

Reply:
Exactly. You can use this function: https://github.com/mozilla/version-control-tools/blob/2c2812d4a41b690203672a183b1dd85ca8b39e01/pylib/mozautomation/mozautomation/commitparser.py#L129.

Reply:
Just copy the function here, making adjustments if necessary, then add a link to the source of the function and say that we can remove it and use the template when https://bugzilla.mozilla.org/show_bug.cgi?id=1528938 is fixed.